shell/common/vsync_waiter.cc

```diff
@@ -114,7 +102,7 @@ void VsyncWaiter::FireCallback(fml::TimePoint frame_start_time,
 
     task_runners_.GetUITaskRunner()->PostTaskForTime(
         [callback, flow_identifier, frame_start_time, frame_target_time]() {
-          FML_TRACE_EVENT("flutter", kVsyncTraceName, "StartTime",
+          FML_TRACE_EVENT("flutter", "VsyncProcessCallback", "StartTime",
                           frame_start_time, "TargetTime", frame_target_time);
           fml::tracing::TraceEventAsyncComplete(
               "flutter", "VsyncSchedulingOverhead", fml::TimePoint::Now(),
```

Comment:
This method call used to report the start time under a VsyncSchedulingOverhead "asynchronous" event name, but if the platform specific waiters will graph the VSYNC events correctly then this event won't be needed any more.

Reply:
I'm okay with removing it in favor of VsyncSchedulingOverhead, but note that VsyncSchedulingOverhead is displayed in another track in the Trace Viewer (see the screenshot in the other thread), so will be a bit harder to see.

Let me know if you still prefer to skip this trace.

Reply:
Unfortunately since the platform waiters can't graph it correctly (see other code comment thread) we should probably keep it.